app/assets/stylesheets/mo/_map_theme_vars_to_bootstrap_vars.scss

```diff
@@ -156,4 +156,4 @@ $panel-default-heading-bg: $LIST_HEADER_BG_COLOR;
 
 $hr-border: $LIST_BORDER_COLOR;
 
-$well-bg: $WELL_BG_COLOR;
```

Comment:
- Add newline at EOF. 
- Unless it messes you up on other projects, I suggest enabling the VSCode `files.insertFinalNewline` setting

Reply:
Thanks. I think these sneaked in while i was trying to set up Rubocop on the other machine.